pkgs/tools/system/dool/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace dool \
       --replace \
```

Comment:
```suggestion
      --replace-fail \
```
This will catch any future changes.

Reply:
I have applied your suggestion, thanks for the review.